src/heaps/mutable_binary_heap.jl

```diff
@@ -251,6 +251,44 @@ function update!(h::MutableBinaryHeap{T}, i::Int, v) where T
         _heap_bubble_down!(comp, nodes, nodemap, nd_id)
     end
 end
+"""
+    pop!{T}(h::MutableBinaryHeap{T}, i::Int)
+
+Pops the element present at index `i` from heap `h` .    
+"""
+function pop!(h::MutableBinaryHeap{T}, i::Int) where T
```

Comment:
This function copies most of its code from `_binary_heap_pop!`, which isn't very DRY.  

You should modify that function to accept `nd_id` as a parameter, defaulting to 1, and then call that function from here.

Reply:
Yes, true!
Pushed a commit addressing your requested change.